backend/src/v5/models/tickets.groups.js

```diff
@@ -37,15 +38,29 @@ Groups.getGroupsByIds = (teamspace, project, model, ticket, groupIds, projection
 	{ teamspace, project, model, ticket, _id: { $in: groupIds } }, projection);
 
 Groups.updateGroup = async (teamspace, project, model, ticket, groupId, data, author) => {
-	const updates = { $set: data };
+	const toUpdate = { };
+	const toUnset = { };
 
-	if (data.rules) {
-		updates.$unset = { objects: 1 };
-	} else if (data.objects) {
-		updates.$unset = { rules: 1 };
+	Object.keys(data).forEach((key) => {
+		if (data[key]) {
```

Comment:
Can we specifically check for null please?

we just had a bug in #4716 that was because we're trying to set something to 0 :'(

So it'd be much easier to check for what we're actually after.

```suggestion
		if (data[key] !== null) {
```